### ISSUE TITLE
Create final release 4.0?

### ISSUE THREAD

**Issue by icemac** [buildout/buildout#673]
I've been successfully using 4.0.0a1 for several weeks now.

I'd like to cut a final release (once #672 is merged).

Are there any objections?

CC: @dataflake @mauritsvanrees

**Comment by dataflake** [comment]
No objections from me.

**Comment by mauritsvanrees** [comment]
I had some more possibly backwards incompatible changes in mind for 4.0.0, like seeing if we can get rid of deprecated calls to `python setup.py`.  But that is not happening any time soon.  So yes, a 4.0.0 would be fine.

**Comment by mauritsvanrees** [comment]
Maybe drop Python 3.8 support after all.  See https://github.com/zopefoundation/meta/pull/291#issuecomment-2550782483
But I don't consider this a blocker.

**Comment by dataflake** [comment]
At this point - feel free.

**Comment by mauritsvanrees** [comment]
@icemac I think a 4.0 release would be good.  There is one draft PR from me from last month, but I am not likely to work on that soon.

**Comment by icemac** [comment]
@mauritsvanrees Which PR do you mean? https://github.com/buildout/buildout/pull/675?

**Comment by mauritsvanrees** [comment]
@icemac Yes, sorry, that is the one I meant.

**Comment by icemac** [comment]
@mauritsvanrees Do we really need https://github.com/buildout/buildout/pull/675 for a final release? I'd not see that PR as a blocker here.
(The previous discussion in this issue was about dropping Python 3.8 support which was done in https://github.com/buildout/buildout/pull/674 and is already merged.)

**Comment by mauritsvanrees** [comment]
@icemac No, we don't need it for a final. My remark was meant to indicate that this is a nice to have, but l see this was a bit between the lines. 😁

**Comment by icemac** [comment]
I just released https://pypi.org/project/zc.buildout/4.0/.

**Comment by mauritsvanrees** [comment]
Thanks!  Right on time for a Plone 6.1 release candidate.
